src/main/java/nextstep/subway/line/application/PathService.java

```diff
@@ -28,47 +27,62 @@ public PathService(StationRepository stationRepository, SectionRepository sectio
     }
 
     public List<StationResponse> getShortestPath(final String sourceStationId, final String targetStationid) {
+        Optional<Station> sourceStation = stationRepository.findById(Long.valueOf(sourceStationId));
+        Optional<Station> targetStation = stationRepository.findById(Long.valueOf(targetStationid));
```

Comment:
```suggestion
        Station sourceStation = stationFind(Long.valueOf(sourceStationId));
        Station targetStation = stationFind(Long.valueOf(targetStationid));
```

반복되는 코드를 메소드로 분리시켜도 좋을것 같네요!